src/content/docs/usage/Animating/procedural.mdx

```diff
@@ -9,125 +9,156 @@ import sophGraphImg from 'assets/bopGraph.png';
 import resVideo from 'assets/opacity.webm';
 import bopVideo from 'assets/bop.webm';
 
-Ah, I see you want to make some cool stuff. Splendid! Don't be scared, procedural animations are easier than they sound.
+This guide introduces procedural animations, explains why they matter, and how to create both simple and complex animations using nodes.
 
 :::tip
-I recommend checking out <a href="/docs/usage/node-graph/getting-started-with-node-graph">node docs</a> and <a href="/docs/usage/animating/getting-started">animating basics</a> first, as it will make understading stuff easier.
+Before diving in, you may find it helpful to review the <a href="/docs/usage/animating/getting-started">Basics of Animations</a> and <a href="/docs/usage/node-graph/getting-started-with-node-graph">Getting Started with Node Graphs</a> pages first.
 :::
 
-## What is a procedural animation
-
-It's an animation that is generated based on ~~prompts~~, ha, no it's not.
-
-Do not confuse procedural animation with generative AI. The effect is 100% based on logic and input parameters, not a black box.
+<br/>
 
-## Crucial animation nodes
+## What is Procedural animation
 
-Without these, you'll have a hard time animating.
+Procedural animation refers to animations that are generated based on logic and input parameters. It's important to not confuse this with generative AI (such as ChatGPT), as those make use of prompts instead. 
 
+<br/>
+## Key animation nodes 
+
+The following nodes are essential when working with procedural animations:
+- **Time node:** Provides the current position in the animation timeline. It exposes:
+  - **Active Frame:** The current frame being displayed.
+  - **Normalized Time;** A value between `0.0` (start of animation) and `1.0` (end of animation).  
+
+  
+  <a style={{textDecoration: "none"}} href="/docs/usage/node-graph/nodes/animation/time"><Node data={{
```

Comment:
Spacing is missing there and bullet points have wrong intendation

<img width="798" height="554" alt="Image" src="https://github.com/user-attachments/assets/5a32ccd9-3cef-4a2f-ae1c-b391d17598d4" />